routes/eventRoutes.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+const express = require('express');
+const router = express.Router();
+const eventController = require('../controllers/eventController');
+const authController = require('./../controllers/authController');
+
+// Protect all routes after this middleware
+router.use(authController.protect);
+// Restrict to admin
+router.use(authController.restrictTo('admin'));
+
+router
+    .route('/connections')
+    .get(eventController.getAllConnectionEvent);
+
+router
+    .route('/connections/:id')
+    .get(eventController.getConnectionEvent);
+
+router
+    .route('/connections/user/:id')
```

Comment:
you could have : /user/:userId/:id like I did for Element if u think is a good idea

Reply:
Didi not see what you are talking about in your pr. Will discuss later